### PR TITLE
Reverse

### PR DESCRIPTION
**Pull request by Moniarchy**

### CODE REVIEW

**Comment by Moniarchy** [comment]
@jrob8577

File: src/reverse.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const reverse = input => {
+	if ( input instanceof Array ){
```
Comment (jrobcodes) [inline src/reverse.js]: Indentation is way off.

File: src/reverse.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	 	return input 
+	 }
+		
+	}
+  
+
+
+export { reverse }
+
+// array !typeof Array ? array :
```
Comment (jrobcodes) [inline src/reverse.js]: Remove unneeded comments and whitespace

**Comment by jrobcodes** [comment]
@Moniarchy @shakalee14 Just a few formatting comments

**Pull request merged**